pkg/fanal/artifact/image/image.go

```diff
@@ -60,6 +66,11 @@ func NewArtifact(img types.Image, c cache.ArtifactCache, opt artifact.Option) (a
 		return nil, xerrors.Errorf("config analyzer group error: %w", err)
 	}
 
+	cacheDir, err := os.MkdirTemp("", "layers")
```

Comment:
I think it's better to use `trivy` in the cache directory name. This is necessary so that users can understand what directory it is and delete it (if necessary)
```suggestion
	layerCacheDir, err := os.MkdirTemp("", "Trivy-layers-cache")
```

Reply:
This is a temporary cache that is removed when Trivy is shutdown